Sources/PackageLoading/Diagnostics.swift

```diff
@@ -77,6 +77,16 @@ extension Diagnostic.Message {
     static func fileReference(path: RelativePath) -> Self {
         .note("found '\(path)'")
     }
+
+    static func infoPlistResourceConflict(
+        path: RelativePath,
+        targetName: String
+    ) -> Self {
+        .error("""
+            resource '\(path)' in target '\(targetName)' is forbidden as it conflicts with the generated resource \
+            bundle Info property list
```

Comment:
We shouldn't talk about the generated resource bundle. How about something along the lines of "Info.plist is not supported as a resource file"

Reply:
I'm worried that may be confusing: Info.plist is not forbidden - it's Info.plist at root which is. Do you have another suggestion?

Reply:
Hmm, "Info.plist is not supported as a top-level resource file in the resources bundle"

Reply:
Updated!